src/OpenLoco/Windows/TextInputWindow.cpp

```diff
@@ -262,7 +262,7 @@ namespace OpenLoco::Ui::TextInput
         }
 
         char* drawnBuffer = (char*)StringManager::getString(StringIds::buffer_2039);
-        strcpy(drawnBuffer, _buffer.data());
+        strcpy(drawnBuffer, _buffer.c_str());
```

Comment:
in general you should never be accessing data() so I went through and corrected all of the mistaken uses.